.github/workflows/build-and-publish.yml

```diff
@@ -3,11 +3,16 @@ name: Build and Publish
 on:
   push:
     paths-ignore:
-    - 'tests/**'
-    branches: [main]
+      - 'tests/**'
+    branches:
+      - main
+      - platform_part_2 # Add this line
```

Comment:
I would remove this. We dont want a build every time you push to this branch. And its already covered by your long running PR.